presto-main/src/main/resources/webapp/index.html

```diff
@@ -128,6 +129,7 @@
                           queryInfo.session.source,
                           queryInfo.session.user,
                           queryInfo.state,
+                          queryInfo.errorType != null ? queryInfo.errorType : "",
```

Comment:
We want to keep this table as succinct as possible. Add a method to shorten the error type:

USER_ERROR -> USER
INTERNAL_ERROR -> INTERNAL
INSUFFICIENT_RESOURCES -> RESOURCES